code/run_fmriprep.sh

```diff
@@ -43,3 +46,6 @@ cmd="singularity run --home $HOME --cleanenv \
 echo Running task ${SGE_TASK_ID}
 echo Commandline: $cmd
 datalad run -d $OUT_DIR -m "Run fMRIPrep on ds002156 ${subject}." $cmd
+
+# Remove any downloaded data in the BIDS dataset to minimize disk usage
+datalad drop -d $BIDS_DIR --what filecontent ${BIDS_DIR}/sub-${subject}
```

Comment:
```suggestion
datalad drop -d $BIDS_DIR --nocheck --if-dirty ignore ${BIDS_DIR}/sub-${subject}
```

Reply:
Will this make sure that, if I modify the BIDS data locally, it won't "drop" those files?